infra/modules/curi/mantarray_downloads/s3.tf

```diff
@@ -0,0 +1,38 @@
+resource "aws_s3_bucket" "log_bucket" {
+  bucket = "curibio-server-access-logs"
+  acl    = "log-delivery-write"
+}
+
+resource "aws_s3_bucket" "downloads" {
+  bucket = var.download_bucket
+  acl    = "public-read"
+
+  policy = jsonencode({
+    Version = "2012-10-17"
+    Statement = [
+      {
+        Sid    = "PublicReadGetObject",
+        Effect = "Allow"
+        Principal = {
+          AWS = "*"
+        }
+        Action   = "s3:GetObject"
+        Resource = "arn:aws:s3:::${var.download_bucket}/*"
+      }
+    ]
+  })
+
+  versioning {
+    enabled = true
+  }
+
+  website {
+    index_document = "index.html"
+    error_document = "error.html"
+  }
+
+  logging {
```

Comment:
consider whether we'd want to just have one "universal" bucket that captures sever access logs from all other buckets. or if it makes sense to segregate them (per prod/modl/test)

Reply:
I think it's fine to separate them by environment